marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/opticPlanExportService.api

```diff
@@ -0,0 +1,37 @@
+{
+    "functionName": "getOpticPlan",
```

Comment:
For consistency, the functionName should be the same  as the filename, minus ".api".

Reply:
Got it.